api/models/node.go

```diff
@@ -9,8 +9,9 @@ type NodeStatus struct {
 }
 
 type ClusterStatus struct {
-	NodeName string         `json:"nodeName"`
-	Members  []cluster.Node `json:"members"`
+	ClusterName string         `json:"clusterName"`
+	NodeName    string         `json:"nodeName"`
+	Members     []cluster.Node `json:"members"`
```

Comment:
I'm assuming that currently the only purpose of this is to make the name queryable via `/cluster`